### PR TITLE
solved using double linked list

### PR DESCRIPTION
**Pull request by Vedika-Agrawal**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I have a smart look  over the guidelines

### Checklist

- [yes ] I've read the contribution guidelines.
- [ yes] I've checked the issue list before deciding what to submit.
- [yes ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#3343 
(Write your answer here.)

### CODE REVIEW

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Vedika-Agrawal As mentioned in the Issue,  you need to check for a word, it means it consists of characters as well not only integer. Therefore, I request you to change the datatype of the input as list of characters.

**Comment by Vedika-Agrawal** [comment]
Sir , Changes are done according to your suggestion. plzz have a look once..

File: C/other/Palindrome.c [hunk 1]
```diff
@@ -1,51 +1,78 @@
...
-C Program to check whether a number is Palindrome or not
-An integer is a palindrome if the reverse of that number is equal to the original number.
-Example of Palindromes are 11,44,101,121,
+Check palindrome words using double linked list in C
+
+Input:
+aabbaa
+
+Output:
+aabbaa is palindrome
```
Comment (siddharth25pandey) [inline C/other/Palindrome.c:8]: Shift it to the end of the Program & Also, write Time and space complexity.
Also, Your Sample Input/Output not matches with the given Program. (Display message is not present)

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Vedika-Agrawal Modifications required

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this PR due to inactivity.

**Pull request closed without merging**